etherman/etherman.go

```diff
@@ -467,24 +754,82 @@ func (etherMan *Client) updateZkevmVersion(ctx context.Context, vLog types.Log,
 	return nil
 }
 
+func (etherMan *Client) updateL1InfoTreeEvent(ctx context.Context, vLog types.Log, blocks *[]Block, blocksOrder *map[common.Hash][]Order) error {
+	log.Debug("UpdateL1InfoTree event detected")
+	globalExitRootL1InfoTree, err := etherMan.GlobalExitRootManager.ParseUpdateL1InfoTree(vLog)
+	if err != nil {
+		return err
+	}
+
+	var gExitRoot GlobalExitRoot
```

Comment:
```suggestion
	 gExitRoot := GlobalExitRoot {
	      MainnetExitRoot: globalExitRootL1InfoTree.MainnetExitRoot,
	      RollupExitRoot: globalExitRootL1InfoTree.RollupExitRoot,
	      BlockNumber: vLog.BlockNumber,
	      GlobalExitRoot: hash(globalExitRootL1InfoTree.MainnetExitRoot, globalExitRootL1InfoTree.RollupExitRoot),
	 }
```

Reply:
that comes from upstream, I'm consciously choosing to reduce code diff over clean code